src/pool.rs

```diff
@@ -80,10 +80,10 @@ where
     /// ```rust
     /// # use sharded_slab::Pool;
     /// let pool: Pool<String> = Pool::new();
-    /// let key = pool.create(|item| item.push_str("Hello")).unwrap();
+    /// let key = pool.create_with(|item| item.push_str("Hello")).unwrap();
     /// assert_eq!(pool.get(key).unwrap(), String::from("Hello"));
     /// ```
-    pub fn create(&self, initializer: impl FnOnce(&mut T)) -> Option<usize> {
+    pub fn create_with(&self, initializer: impl FnOnce(&mut T)) -> Option<usize> {
         let tid = Tid::<C>::current();
         let mut init = Some(initializer);
         test_println!("pool: create {:?}", tid);
```

Comment:
if you decide to continue calling this `create_with`, should probably update the test println here
```suggestion
        test_println!("pool: create_with {:?}", tid);
```